packages/xo-web/src/xo-app/user/index.js

```diff
@@ -115,11 +116,11 @@ class DefaultFilterPicker extends Component {
 
   _handleDefaultFilter = value => setDefaultHomeFilter(this.props.type, value && value.value).catch(noop)
 
-  componentWillMount() {
+  UNSAFE_componentWillMount() {
     this._computeOptions(this.props)
   }
 
-  componentWillReceiveProps(props) {
+  UNSAFE_componentWillReceiveProps(props) {
```

Comment:
Avoid unrelated changes and ignore linting errors on existing code in xo-web 5 :slightly_smiling_face:

Reply:
@MathieuRA Without this change the linter don't let me commit

@julien-f Ok, I will revert the change.

But I think it would be preferable to change the linter config to not prevent the commit in this case. 

Personally, I really don't like the idea to remember and/or take the decision on what should be ignored or not each time I want to commit some changes ^^

Reply:
I agree, but I don't know how to enable the linter only for new changes in xo-web :grimacing: